packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -163,6 +165,18 @@ class AmplifyDataStorePlugin : FlutterPlugin, MethodCallHandler {
 
         val dataStoreConfigurationBuilder = DataStoreConfiguration.builder()
 
+        if(request["hasCustomErrorHandler"] as Boolean) {
```

Comment:
I think this naming `hasCustomErrorHandler` is inconsistent and should just be `hasErrorHandler`